src/core/contexts.ts

```diff
@@ -25,7 +25,8 @@ export type StoreForDevelopment = [
     atoms: AnyAtom[]
     state: State
     listeners: Set<() => void>
-  }>
+  }>,
+  restoreAtoms: (values: Iterable<readonly [AnyAtom, unknown]>) => void
```

Comment:
```suggestion
  restore: (values: Iterable<readonly [AnyAtom, unknown]>) => void
```
just to be consistent.